src/components/pages/unmarkedWinterStoragePage/unmarkedWinterStoragePage.scss

```diff
@@ -0,0 +1,13 @@
+.vene-unmarked-winter-storage-page {
+  &__buttons {
+    display: flex;
+    flex-direction: row;
```

Comment:
Style has no effect since `row` is default value. Fun fact: in React Native it's `column` :)